core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -0,0 +1,106 @@
+package uk.ac.bbsrc.tgac.miso.core.service.printing;
+
+import java.io.UnsupportedEncodingException;
+
+import org.apache.commons.codec.binary.Base64;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Barcodable;
+import uk.ac.bbsrc.tgac.miso.core.util.LimsUtils;
+
+public enum Driver {
```

Comment:
Javadoc on class, `encode` and elements would be helpful. I'm assuming the elements refer to specific devices?